dev-client/src/components/SoilPropertiesDataTable.tsx

```diff
@@ -84,38 +85,40 @@ export const SoilPropertiesDataTable = ({rows, ...containerProps}: Props) => {
   };
 
   return (
-    <Box {...containerProps}>
-      <Row justifyContent="flex-start">
-        <DataTableHeader
-          width={columnWidthDepth}
-          text={t('site.soil_id.site_data.soil_properties.depth', {
-            units: 'METRIC',
-          })}
-        />
-        <DataTableHeader
-          width={columnWidthTexture}
-          text={t('site.soil_id.site_data.soil_properties.texture')}
-        />
-        <DataTableHeader
-          width={columnWidthColor}
-          text={t('site.soil_id.site_data.soil_properties.color')}
-        />
-        <DataTableHeader
-          width={columnWidthRockFragment}
-          text={t('site.soil_id.site_data.soil_properties.rock_fragment')}
-        />
-      </Row>
+    <ScrollView horizontal={true}>
```

Comment:
After discussing with @knipec we agreed to pull the `ScrollView` into the table component, since that was its only use-case in practice. We also agreed to factor out the `containerProps` property because it was not used anywhere.